client/my-sites/plans-grid/types.ts

```diff
@@ -8,6 +8,8 @@ export type TransformedFeatureObject = FeatureObject & {
 };
 
 export interface PlanActionOverrides {
+	canUserManageCurrentPlan: boolean | null;
+	currentPlanManageHref: string | null;
```

Comment:
Thanks for thinking about this. prop-drilling is experience-killing :D

Something doesn't feel right with these sitting here though (at the top level at least). `PlanActionOverrides` was meant to have a clear semantic, holding a set of props for overriding the CTAs on the page (not a placeholder for relevant helpers). It might not be a super polished approach though 🤷‍♂️

For action overrides I'd expect a description like the following: 

> Used for overwriting the action button properties for a respective context and plan e.g. `loggedInFreePlan` used to customise the free-plan CTA in a  logged-in/user session. An action override contains a `callback` (the event handler to override), `text` (the text to display), `status` (not sure how to explain this exactly)... etc.

Before we delve any further, I have a quick idea that might resonate: How about we create a `currentPlan` override, and pass the relevant data (href and text) instead? so we won't need to pass `canUserManageCurrentPlan` and `currentPlanManageHref` into the grid at all, makes sense at all?

Reply:
>Something doesn't feel right with these sitting here though (at the top level at least)

Thanks @chriskmnds I wanted to spark some conversation here, had the same feeling myself.

>  How about we create a currentPlan override

This is perfect! 😁